actors/account/src/lib.rs

```diff
@@ -118,6 +118,6 @@ impl ActorCode for Actor {
         Constructor => constructor,
         PubkeyAddress => pubkey_address,
         AuthenticateMessageExported => authenticate_message,
-        _ => fallback [raw],
+        _ => fallback,
```

Comment:
`[raw]` is no longer necessary as the dispatch logic "picks" the correct behavior based on the method's signature.